README.md

```diff
@@ -238,3 +238,41 @@ pattern.
 [pytest]: https://docs.pytest.org/en/latest/
 [mozilla]: https://github.com/mozilla/addons-server
 [selenium-training]: https://qualitas-server.herokuapp.com/wiki/Introduction_to_Pytest_and_Selenium
+
+### HOW TO CREATE TEST DATA FOR REDIRECTS
```

Comment:
Sorry,  I should have caught this the first time, but I just now looked at it in rich diff form: In terms of structure should this be ## vs ###? In other words, is this section a child of the "Framework Design" section or a peer?

Also, just a minor comment, but perhaps it would be nice to line up the styling with the other sections so they match a bit better (?) (this is a more subjective point, I know)

Reply:
updated the title height with ##